features/src/vip-cli/devcontainer-feature.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "vip-cli",
     "name": "VIP CLI",
-    "version": "1.3.0",
+    "version": "1.4.0",
     "description": "Installs VIP CLI into the Dev Environment",
```

Comment:
Because "no good deed goes unpunished".... 
Would it be possible to update "VIP CLI" to "VIP-CLI"? 🙈

Reply:
Sure

Reply:
Spectacular, thank you!